Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h

```diff
@@ -827,9 +840,11 @@ class Variational_shape_approximation {
     using parameters::choose_parameter;
 
     const FT subdivision_ratio = choose_parameter(get_parameter(np, internal_np::subdivision_ratio), FT(5.0));
+    const FT boundary_subdivision_ratio = choose_parameter(get_parameter(np, internal_np::boundary_subdivision_ratio), FT(5.0));
```

Comment:
In order to be backward compatible we should set `boundary_subdivision_ratio`to the value of `subdivision_ratio` as default.

Reply:
@sloriot.  Is this a good idea?  This somehow assumes an order on named parameters.

Reply:
This is correct, the variable `subdivision_ratio` being initialized before the call to the next get param.